Makefile

```diff
@@ -38,6 +38,15 @@ hook-gitter:
 	@curl -s -X POST -H "Content-Type: application/json" -d "{\"payload\":`curl -s -H "Accept: application/json" https://circleci.com/api/v1/project/goldmann/docker-scripts/${CIRCLE_BUILD_NUM}`}" ${GITTER_WEBHOOK_URL}
 
 release: clean
-	python setup.py clean
-	python setup.py sdist
-	twine upload dist/*
+	pip install zest.releaser
```

Comment:
This will not fly. We need to make sure things are in proper virtualenvs.

Reply:
Good point, moved to Pipfile.